udata/features/territories/commands.py

```diff
@@ -12,12 +12,17 @@
 
 import requests
 
-from udata.models import (
-    Dataset, TERRITORY_DATASETS, ResourceBasedTerritoryDataset
-)
+from udata.app import create_app
+from udata.models import Dataset, ResourceBasedTerritoryDataset
 from udata.commands import submanager
 from udata.core.storages import logos, references, tmp
 
+
+app = create_app()
```

Comment:
You shouldn't. The app is already created by the script launcher: https://github.com/etalab/udata/blob/master/udata/commands/__init__.py#L72-L78
You should use `current_app.config` to access the config within a command.